doc/conf.py

```diff
@@ -338,6 +338,11 @@
     'compound': (_python_doc_base + '/reference/compound_stmts.html#%s', ''),
 }
 
+if 'dev' in release:
```

Comment:
That's true but we also don't want that our binder notebooks in the stable docs to be created from main. What I assumed we could do is checkout the stable version by using a tag, but I'm not sure if this is the correct way to do it.

Reply:
You're right, we don't want it to always be set to `main`. 
I believe you made the right choice by using the version tag since the URL should resolve in the same way : 

https://github.com/sphinx-gallery/sphinx-gallery/blob/18705f8948008df84bf6c6b38df4d88d701d3979/sphinx_gallery/binder.py#L64-L74

Reply:
Ok great thanks for confirming!